<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YubiKeyTester</class>
 <widget class="QMainWindow" name="YubiKeyTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>38015</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PassMan YubiKey Tester</string>
  </property>
  <widget class="QWidget" name="yubiKeyCentralWidget">
   <widget class="QLabel" name="challengeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Challenge:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="challengeLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="responseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HMAC-SHA1 Response:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="responseLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="configSlotLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration Slot:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="slotOneRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>One</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="slotTwoRadioButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Two</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Initiate Challenge</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="serialNumberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="versionLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="versionLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Version:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
